src/vl53l4cx_def.h

```diff
@@ -31,7 +31,8 @@ extern "C" {
  *  @{
  */
 
-
+/** VL53L4CX IMPLEMENTATION model id */
+#define VL53L4CX_IDENTIFICATION_MODEL_ID       ((uint16_t)0x010F)
```

Comment:
This define is useless, because you can use the define "VL53L4CX_IDENTIFICATION__MODEL_ID" that you can find in vl53l4cx_register_map.h.

Reply:
Thanks for all your responses and clarifications!

I must have been referencing an older revision of the datasheet.

Based on my previous assumption, I was assuming the Sensor ID of the device (Model_ID and Module_Type) would uniquely identify which sensor is in use.

Since the VL53L4CX and VL53L4CD sensors have the same Model_ID and Module_Type, is there a way to query the device to determine which sensor it is?

I will update the PR. I will also change the name of the proposed function from VL53L4CX_GetModelId to VL53L4CX_GetSensorId, akin to the function VL53L4CD_GetSensorId for the VL53L4CD sensor.